src/helpers/contextCreatorHelpers/index.js

```diff
@@ -15,6 +15,16 @@ export const addErrorsAndDataToManifest = ({ manifest, errors, data }) => ({
   }),
 });
 
+export const addErrorsAndDataToManifestKeyValuePair = ({ manifest, errors, data }) => ({
+  ...manifest,
+  questions: Object.entries(manifest.questions).map((question) => {
+    const modifiedQuestion = { ...question[1] };
```

Comment:
why the `[1]`?

Reply:
it breaks it down into a weird keyvalue pair
with ['0', {object)] so needs the 1 for some reason?

Reply:
oooh yes ok you can do a short cut here

`Object.entries(manifest.questions).map(([questionKey, questionValue]) => {`

I think that's the syntax.. basically will destruture it and you can use the questionValue instead of question[1]

Reply:
yeah that works but then it complains the questionValue isn't getting used may just have to stick to the [1] bit